src/components/Button/Button.css

```diff
@@ -0,0 +1,14 @@
+.button {
+  width: 160px;
+  height: 42px;
+  background-color: #FAA805;
+  border-radius: 6px;
+  margin: 0;
+  padding: 0;
+  border: none;
+  font-family: Inter, Arial, Helvetica, sans-serif;
```

Comment:
could we set font-family for the entire page the same, or does button have a unique font? @kvnloughead

Reply:
You're right, I think pretty much everything has Inter font-family, I'll change it.